Cargo.toml

```diff
@@ -10,10 +10,14 @@ categories = ["cryptography", "no-std"]
 keywords = ["miden", "crypto", "hash", "merkle"]
 edition = "2021"
 
+[features]
+default = ["std", "winter_crypto/default", "winter_math/default", "winter_utils/default"]
```

Comment:
I think its friendlier to extend the default features request upstream.

We happen to have only `std` in `winter_*`, but if they someday introduce some default feature different than `std`, we don't need to bump our crate to get this feature - it will be included automatically.

Given `default-features` is a general property of the language, and not an implementation detail of the libraries, I would see as desirable to extend it upstream, and that our downstream dependents expects the same from us.

wdyt?

Reply:
Makes sense! This is not how we do it in other crates - so, maybe we should create an issue (or maybe even just a PR) to update `Cargo.toml` files for all Miden crates to follow this pattern?